Source: Steve Tell (http://lists.gnu.org/archive/html/bug-gnu-utils/2003-12/msg00023.html)
Upstream: Sent in 2003, not applied yet.
Reason: Gentoo bug #47688
--- plotutils-2.5.1.orig/graph/plotter.c	2008-09-26 23:48:15.000000000 +0200
+++ plotutils-2.5.1/graph/plotter.c	2008-11-08 13:10:14.000000000 +0100
@@ -1011,7 +1011,7 @@
   multigrapher->x_trans.output_min = 0.0;
   multigrapher->x_trans.output_max = (double)PLOT_SIZE;
   multigrapher->x_trans.output_range = multigrapher->x_trans.output_max - multigrapher->x_trans.output_min;
-  multigrapher->x_trans.output_min = 0.0;
+  multigrapher->y_trans.output_min = 0.0;
   multigrapher->y_trans.output_max = (double)PLOT_SIZE;
   multigrapher->y_trans.output_range = multigrapher->y_trans.output_max - multigrapher->y_trans.output_min;
 
